modules/builds-strategy-docker-entitled-satellite.adoc

```diff
@@ -33,3 +37,4 @@ RUN rm /etc/rhsm-host && \
 USER 1001
 ENTRYPOINT ["/bin/bash"]
 ----
+<1> If adding Satellite configurations to builds using `enabled=1` fails, add `RUN sed -i".org" -e "s#^enabled=1#enabled=0#g" /etc/yum/pluginconf.d/subscription-manager.conf` to the Dockerfile.
```

Comment:
s/Satellite/Red Hat Satellite

Reply:
Thanks! Updated the modules in this assembly to each use Red Hat Satellite on first reference just in case they're reused.